ecmascript/parser/Cargo.toml

```diff
@@ -42,4 +42,4 @@ pretty_assertions = "0.6"
 name = "lexer"
 
 [[example]]
-name = "typescript"
\ No newline at end of file
+name = "typescript"
```

Comment:
If you are using windows, please set autocrlf and change line endings to lf

Reply:
no, it's vs code on a mac

Reply:
it just added a new line to a file that didn't have an empty last line

Reply:
I got it.